.github/workflows/run-unittest.yml

```diff
@@ -0,0 +1,26 @@
+name: CBI unittest
+
+on: [push]
```

Comment:
The equivalent file in the [P3 Analysis Library](https://github.com/intel/p3-analysis-library/blob/main/.github/workflows/build-unittest.yml) CI also runs on pull requests.  Do we need both here too?

Reply:
Addressed in d1c985f